snuba/reader.py

```diff
@@ -22,12 +31,14 @@
 else:
     Result = MutableMapping[str, Any]
 
+TQueryType = TypeVar("TQueryType")
```

Comment:
Isn't `-Type` redundant here — doesn't the `T-` prefix stand for that?

Reply:
Just an old habit of prefixing generic parameters with T to distinguish them from actual types. It was common in hacklang and java. I am fine removing it since here they are actually types we define.

Reply:
I like the `T-` prefix rather than the `-Type` one for generics, personally, for what it's worth. I think `TQuery` reads better than `QueryType` and it's clearer (to me at least) that it's a type variable rather than a concrete type.

Reply:
ahhh you were talking about Type at the end, not the T at the beginning, I read it wrong. yeah, will remove that